ucte/ucte-converter/src/test/resources/frVoltageRegulatingXnodeReport.json

```diff
@@ -0,0 +1,243 @@
+{
+  "version" : "1.0",
+  "reportTree" : {
+    "taskKey" : "rootTaskKey",
+    "childReporters" : [ {
+      "taskKey" : "ImportUcteData",
+      "childReporters" : [ {
+        "taskKey" : "UcteReading",
+        "childReporters" : [ {
+          "taskKey" : "fixUcteNodes"
+        }, {
+          "taskKey" : "fixUcteLines",
+          "reports" : [ {
+            "reportKey" : "negativeLineResistance",
+            "values" : {
+              "reportSeverity" : {
+                "value" : "UCTE_ERROR",
+                "type" : "ERROR_LOGLEVEL"
+              },
+              "lineId" : {
+                "value" : "FFFFFF11 XXXXXX11 1",
+                "type" : "UNTYPED"
```

Comment:
Maybe we should just not write the type in this case?

Reply:
Yes, indeed, I wanted to still add a Serializer/Deserializer for TypedValue to that end. I thought to either not write the type or directly write the value in fact.